portfolio/src/main/java/com/google/sps/servlets/ServeCommentsServlet.java

```diff
@@ -0,0 +1,91 @@
+// Copyright 2019 Google LLC
```

Comment:
nit: 2020?

Reply:
Isn't it version of copyright? I'm not sure.

Reply:
See go/copyright#the-year

Reply:
Done.